src/pages/dashboard/analysis/components/Charts/WaterWave/index.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { Component } from 'react';
+
+import autoHeight from '../autoHeight';
+import styles from './index.less';
+
+/* eslint no-return-assign: 0 */
+/* eslint no-mixed-operators: 0 */
+// riddle: https://riddle.alibaba-inc.com/riddles/2d9a4b90
+
+export interface WaterWaveProps {
+  title: React.ReactNode;
+  color?: string;
+  height?: number;
+  percent: number;
+  style?: React.CSSProperties;
+}
+
+class WaterWave extends Component<WaterWaveProps> {
+  state = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'state' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)